### DISCUSSION TITLE
IBAX PROJESİ HAKKINDA YORUMLAMALAR (Herkes katılım göstersin lütfen)

### DISCUSSION THREAD

**Discussion by Musaix** [IBAX-io/go-ibax#983] [Soru Cevap - Türkçe]
Proje hakkında yaptığım araştırmalara dayanarak güzel kazançlar elde edeceğimizi düşünüyorum. Maks Arz miktarı yüksek olsa da kendi ağı ve cüzdanını kullancağı için ileride güzel noktalara gelecek gibi duruyor. Sizler neler düşünüyorsunuz ?

**Comment by dervishh** [comment]
Ciddi bir beklenti oluştu. Fazlası ile karşılayacağını düşünüyorum..

**Comment by Yusufknk** [comment]
İlerleyen zamanlarda yakım olacak

**Comment by papi232306** [comment]
beklenti yüksek. proje uumut verici

**Comment by feromen06** [comment]
Beni encok heyecanlandıran şey. Yazılım bilmeyenlerin bile kolaylıkla token çıkarabilmesi. Bu özellik tutarsa Sibel Can bile gelip token cikaririr :))
